tests/unit/network_graph/test_graph.py

```diff
@@ -346,7 +349,10 @@ def test_capacity_path_single_hop_with_fees(complex_community_with_trustlines_an
     source = A
     destination = B
 
-    sendable, max_path = complex_community_with_trustlines_and_fees.find_maximum_capacity_path(
+    (
```

Comment:
What's up with black

Reply:
I guess they changes the rules. I updated black and that is what black then did to the code.